### ISSUE TITLE
 Audit #1 – Login Page – Focus styles on text input missing

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4322]
Connected to Audit 1, see #4300 
1.1 Login Page, URL: https://ocis-a11y.owncloud.works/

# Issues

Text Inputs have no focus styles 

# Remediations
For example changing outline color on focus.

# Best practice/tip
Avoid unneccessary animations
